Xamarin.Forms.Platform.Android/Renderers/FormsVideoView.cs

```
@@ -0,0 +1,103 @@
+using Android.Content;
+using Android.Media;
+using Android.Runtime;
+using Android.Widget;
+using System;
+using System.Collections.Generic;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public class FormsVideoView : VideoView
+	{
+		int _videoHeight, _videoWidth;
+
+		public FormsVideoView(Context context) : base(context) { }
+
+		public override void SetVideoPath(string path)
+		{
+			base.SetVideoPath(path);
+
+			if (System.IO.File.Exists(path))
+			{
+				MediaMetadataRetriever retriever = new MediaMetadataRetriever();
+				try
+				{
+					retriever.SetDataSource(path);
+					ExtractMetadata(retriever);
+				}
+				catch { }
```

We can't do this and just swallow the error.